### ISSUE TITLE
508 issues migration from vets.gov -> va.gov

### ISSUE THREAD

**Issue by andreahewitt-odd** [department-of-veterans-affairs/va.gov-team#846]
## User Story
As an assessability auditor, I need all the open 508 issues moved from the vets.gov repo to the va.gov repo so I can assign them out to teams to fix.

## Goal
_All open 508 issues moved to from the vets.gov repo to the va.gov repo_

## Acceptance Criteria
- [ ] _There are no open 508 audit issues in the vets.gov repo_

## Definition of Done
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Practice%20Areas/Accessibility/508officeReviewList.md

**Comment by 1Copenut** [comment]
1. Old versions of 508 issues are in the [vets.gov-team repo](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Practice%20Areas/Accessibility/508officeReviewList.md), which is private, so we'll have to copy these manually.
2. Create a new issue in the va.gov-team repo using our [508 issue template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=508%2FAccessibility&template=508-issue.md&title=).
3. Add the `triage-incident` and `sev-1 | 2 | 3 ` labels to the ticket.
4. Assign to Alex on the Triage team.
5. Copy and paste the old issue description into the new issue template. There's a three-dot menu on the upper right of an issue detail view that lets you edit the raw Markdown. This is the info you want to copy/paste to retain formatting.
6. Please scrub any mention of test user credentials from the tickets before you post them to va.gov-team.
7. Preview the ticket, and submit it.
8. Copy the ticket URL, and add it to our new 508 product list by pressing the Edit pencil and updating the Markdown.
9. Only copy open issues. I've included a link back to the previous document for older closed items.
10. When all the issues for a row have been copied to our new document, please add a **COPIED** text to the old document, so we don't duplicate each other's effort.

**Comment by 1Copenut** [comment]
@andreahewitt-odd && @peterhill-oddball 
I'm waiting on one design-ish issue for the design team to weigh in. Will add it if necessary, or close it if not. Either way, we can consider the 508 migration of tickets done.

Thank you both so much for the help closing this out!

**Comment by peterhill-oddball** [comment]
> @andreahewitt-odd && @peterhill-oddball
> I'm waiting on one design-ish issue for the design team to weigh in. Will add it if necessary, or close it if not. Either way, we can consider the 508 migration of tickets done.
> 
> Thank you both so much for the help closing this out!

Huzzah -- thanks for the guidance!
